### ISSUE TITLE
Simple step-by-step guide for DebugServerConsole

### ISSUE THREAD

**Issue by markbirss** [c-sky/forum#73]
Hi

From the video it not clear howto use DebugServerConsole

http://action.news/watch?v=Y9gA1q9NA_Y

It would help just to show the host and device commands a bit more clearly

**Comment by minkione** [comment]
I agree. The product looks interesting but NO English documentation...
Can someone work on the documentation part?

**Comment by markbirss** [comment]
@minkione 

I was able to find what might help here with regard to instructions for DebugServerConsole

https://gitlab.savoirfairelinux.com/netdsa/buildroot/commit/6a4131d101cf0d8bc35ce15672524f144c925fc3#39c9f39fa5b9ea0be6bf944be187f42d8eea0188
